lib/datadog/appsec/contrib/rack/reactive/request.rb

```diff
@@ -9,29 +11,29 @@ module Rack
         module Reactive
           # Dispatch data from a Rack request to the WAF context
           module Request
+            WAF_ADDRESSES = [
```

Comment:
No need to create a new array for every time we call `self.subscribe`

Reply:
I would drop the `WAF_` prefix here. Indeed currently only the WAF is called, but it may not be so in the future as some processing parts that would react to these addresses may be implemented outside of libddwaf.

In fact, the names are different from what you can see in `waf_args` (for which the keys are the WAF addresses) _on purpose_.

Reply:
Done